camel/types/enums.py

```diff
@@ -338,6 +352,15 @@ def token_limit(self) -> int:
             ModelType.GLM_3_TURBO,
             ModelType.GLM_4,
             ModelType.QWEN_VL_PLUS,
+            ModelType.NVIDIA_LLAMA3_CHATQA_70B,
+            ModelType.NVIDIA_LLAMA3_CHATQA_8B,
+            ModelType.NVIDIA_NEMOTRON_340B,
+            ModelType.NVIDIA_YI_LARGE,
+            ModelType.NVIDIA_MISTRAL_LARGE,
+            ModelType.NVIDIA_LLAMA3_70B,
+            ModelType.NVIDIA_LLAMA2_70B,
+            ModelType.NVIDIA_MIXTRAL_8X7B,
+            ModelType.NVIDIA_DRACARYS_LLAMA31_70B,
         }:
             return 8_192
```

Comment:
I think not all nvidia models has token limit as `8192`, could you double check this?

Reply:
Thanks Wendong, just updated the token limit for models